common.blocks/i-bem/__collection/i-bem__collection.js

```diff
@@ -164,7 +164,7 @@ var BemCollection = inherit(/** @lends BemCollection.prototype */{
     /**
      * Returns a new collection comprised of collection on which it is called joined with
      * the collection(s) and/or array(s) and/or entity(es) provided as arguments.
-     * @param {Collection|Array|BemEntity} [args...]
+     * @param {?...(Collection|Array|BemEntity)} args
```

Comment:
Not exactly. The difference is in `arguments.length`. 

For example, the first argument of most nodejs' callback functions is an error object, which is the nullable argument (it can be `null`, but the argument must be passed to the function). "Optional argument" means that it can be omitted, but if it presents in must has a proper type.

~~~
› [1].concat()
‹ [1]
› [1].concat(null)
‹ [1, null]
› [1].concat(undefined)
‹ [1, undefined]
~~~